Wox/App.xaml.cs

```diff
@@ -60,10 +60,7 @@ private void OnStartup(object sender, StartupEventArgs e)
                 AutoStartup();
                 AutoUpdates();
 
-                if (!_settings.HideOnStartup)
-                {
-                    mainVM.MainWindowVisibility = Visibility.Visible;
-                }
+                mainVM.MainWindowVisibility = _settings.HideOnStartup ? Visibility.Hidden : Visibility.Visible;
```

Comment:
If I put a breakpoint on `if (!_settings.HideOnStartup)` `mainVM.MainWindowVisibility` was already `Visibility.Visible` by default, which results in still showing the Wox window after startup when "Hide Wox on startup" is selected. I experienced this on two pc's both running Windows 10